doc/manuals/orion-api.md

```diff
@@ -3892,6 +3892,38 @@ Example:
 }
 ```
 
+Example using geoquery as condition:
+
+```json
+{
+  "description": "One subscription to rule them all",
+  "subject": {
+    "entities": [
+      {
+        "idPattern": ".*",
+        "type": "Room"
+      }
+    ],
+    "condition": {
+      "attrs": [ "temperature" ],
+      "expression": {
+        "georel": "near;maxDistance:15000",
+        "geometry": "point",
+        "coords": "coords": "37.407804, -6.004552"
+      }
+    }
+  },
+  "notification": {
+    "http": {
+      "url": "http://localhost:1234"
+    },
+    "attrs": ["temperature", "humidity"]
+  },            
+  "expires": "2025-04-05T14:00:00.00Z",
+  "throttling": 5
```

Comment:
```suggestion
  "expires": "2025-04-05T14:00:00.00Z"
```

Remove also in L3891.

Reply:
Done at https://github.com/telefonicaid/fiware-orion/pull/4431/commits/e56a7be2ba768229a7cbbe6e4e8a91f1feefea02